client/src/components/Slider/Slider.css

```diff
@@ -0,0 +1,47 @@
+.slide {
+  height: 280px;
+  width: 280px;
+  margin-right: 30px;
+}
+
+.title {
+  width: 340px;
+  background-color: rgb(246, 230, 222);
+  text-align: left;
+  font-weight: bold;
+}
+
+.card {
+  display: flex;
+}
+
+.slide-container {
+  display: flex;
+  flex-direction: row;
+  height: 260px;
+  width: 320px;
+  background-color: rgb(246, 230, 222);
+  border-left: 20px solid rgb(246, 230, 222);
+  margin-right: 0;
+}
+
+.button {
+  background-color: rgb(246, 230, 222);
+  border: 0;
+  margin: 0;
+  font-size: 20px;
+  height: 278px;
+}
+
+img {
```

Comment:
Can you use classes here? These are going to clash with styles from other components

Reply:
done